lib/lowkiq.rb

```diff
@@ -53,9 +53,9 @@ def client_redis_pool
       @client_redis_pool ||= ConnectionPool.new(size: client_pool_size, timeout: pool_timeout, &redis)
     end
 
-    def middleware_wrapper(middlewares)
+    def server_wrapper
```

Comment:
You misunderstand me.

You should keep `middleware_wrapper`, `client_wrapper` and add `server_wrapper` like `client_wrapper`.

```ruby
def client_wrapper
      @client_wrapper ||= self.middleware_wrapper(self.client_middlewares)
end
```

and also add 

```ruby
def server_wrapper
      @server_wrapper ||= self.middleware_wrapper(self.server_middlewares)
end
```

Reply:
Sorry, I misunderstood. I have corrected it. You can review it.